### ISSUE TITLE
problem when installing in windows

### ISSUE THREAD

**Issue by swethasharma** [deanmalmgren/textract#108]
Hi,

  I am trying to install textract in windows.I installed with pip command, the program exited with the following error

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Users\swetha\AppData\Local\Temp\pip-build-9a7u8uyb\pdfminer\setup
```

.py", line 3, in <module>
        from pdfminer import **version**
      File "C:\Users\swetha\AppData\Local\Temp\pip-build-9a7u8uyb\pdfminer\pdfmi
ner__init__.py", line 5
        print **version**
                        ^
    SyntaxError: Missing parentheses in call to 'print'

```
----------------------------------------
```

Command "python setup.py egg_info" failed with error code 1 in C:\Users\swetha\A
ppData\Local\Temp\pip-build-9a7u8uyb\pdfminer\

How do i fix this error?
-swetha

**Comment by bjgordon** [comment]
I'm having the same issue with python 3.5 on windows. It looks like Python 3 isn't supported based on this PR waiting to be merged. https://github.com/deanmalmgren/textract/pull/104

**Comment by deanmalmgren** [comment]
This particular problem looks like its related to pdfminer not being python 3 compatible.

Sorry for the hold-up on the python 3 integration. The lack of backwards compatibility with some packages is giving me pause although it does seem like there are some options out there. If you try @sirex's python 3 branch, does that resolve the issue you're having @bjgordon @swethasharma?

**Comment by swethasharma** [comment]
Thanks for the response.I have to try it.will keep you posted about this.
-swetha

**Comment by bjgordon** [comment]
The install still failed for me, but it was a problem installing EbookLib-0.15. Attached the console output if you're interested, but I guess I'll have to get that dependency working first.
[sirex install log.txt](https://github.com/deanmalmgren/textract/files/246269/sirex.install.log.txt)

**Comment by bjgordon** [comment]
I commented on the other thread, FYI @swethasharma I had to use master of Ebook-Lib, 0.15 didnt work.

**Comment by deanmalmgren** [comment]
It would be great to get a PR that adds windows installation instructions. There are some ideas for getting started in #111. I'm going to close out this issue while I'm decluttering a bit.

As for the issues with python3, that is close to being resolved by #126 + #104 + #116.
